conans/client/loader.py

```diff
@@ -103,6 +106,9 @@ def load_conan(self, conan_file_path, output, consumer=False):
             result = self._create_check_conan(loaded, consumer, conan_file_path, output, filename)
             if consumer:
                 result.options.initialize_upstream(self._options)
+                result.scope = self._scopes[None]
```

Comment:
scope[None] is the set of scopes for the "root" node of the graph, the consumer. It is None as it might not have a name, as opposed as normal packages that they must have a name.